kubernetes/config-skeleton.libsonnet

```diff
@@ -281,5 +281,24 @@
         'app.kubernetes.io/part-of': 'aproplan',
       },
     },
+    kedaScalerConfig: {
+      name: 'scaler-%s' % s.deployment.name,
+      trigerConfigs: [],
+      behavior: s.common.autoscaling.behavior,
+      fallback: {
+        failureThreshold: 3,
+        replicas: 2,
+      },
+      scaleTarget: {
```

Comment:
This should be configurable. We scale both statefulsets and deployments

Reply:
Done